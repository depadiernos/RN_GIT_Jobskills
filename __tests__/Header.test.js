import React from "react";
import {Platform} from 'react-native';
import { create } from "react-test-renderer";
import SearchHeader from 'react-native-search-header';

function SearchHeaderComponent(props) {
    return <SearchHeader
        topOffset={Platform.OS == "ios" ? 24 : 0}
        ref={(searchHeader) => {
            this.searchHeader = searchHeader;
        }}

        placeholder='Search Job..'
        enableSuggestion={false}
        placeholderColor='gray'
        headerHeight={56}
        pinnedSuggestions={[`react-native-search-header`, `react-native`, `javascript`]}
        onClear={() => {
            console.log(`Clearing input!`);
            this.SearchFilterFunction('')
        }}
        onEnteringSearch={({ nativeEvent }) => this.SearchFilterFunction(nativeEvent.text)}
    />
}

describe("FlatList component", () => {
    test("Matches the snapshot", () => {
        const button = create(<SearchHeaderComponent />);
        expect(button.toJSON()).toMatchSnapshot();
    });
});