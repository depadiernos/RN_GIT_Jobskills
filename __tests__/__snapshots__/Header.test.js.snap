// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlatList component Matches the snapshot 1`] = `
<View
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "transparent",
      "elevation": 2,
      "justifyContent": "flex-start",
      "overflow": "hidden",
      "position": "absolute",
      "shadowColor": "#576970",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.25,
      "shadowRadius": 2,
      "top": 24,
      "transform": Array [
        Object {
          "translateX": -750,
        },
      ],
      "width": "100%",
      "zIndex": 10,
    }
  }
/>
`;
